src/lib_python/ioda_conv_engines.py

```diff
@@ -96,7 +96,7 @@ def WriteGeoVars(self, GeoVars, GeoVarDims, GeoVarAttrs):
                 dims = GeoVarDims[VarName]
             else:
                 # assume it is just nlocs
-                dims = ['Location']
+                dims = ['nlocs']
```

Comment:
My fault!! I missed this one.

Reply:
ok,, this is odd to me is this for backward support?

Reply:
the geoval reader for the UFO ctests still assumes 'nlocs' (I believe) and not Location

Reply:
ah yes,, thanks